### ISSUE TITLE
Given mount not allowed

### ISSUE THREAD

**Issue by enggrey** [joernhees/userdocker#12]
Hi, I'm having difficulties sharing folders with the containers. I keep getting 

> ERROR: given mount not allowed: /home/files:/project/files

Using docker or nvidia-docker this is no problem, so I guess this has to do with the permissions in the config file. This is how the relevant part looks like at the moment.

```
VOLUME_MOUNTS_ALWAYS = []
VOLUME_MOUNTS_AVAILABLE = ['/home/files:/project/files']
VOLUME_MOUNTS_DEFAULT = [
    # default mount user's home
    user_home + ':' + user_home,
]
```

Do you see any obvious mistakes? 

Thanks in advance!

**Comment by joernhees** [comment]
hmm, cannot reproduce, could you paste the commandline you're trying?

**Comment by joernhees** [comment]
might've been related to #13 and possibly also fixed by 5ffb9b6edc00debd91a1f4739381fe1bccd33752

closing for now, reopen if not
